Migrations/Mysql/Version20250721210441.php

```
@@ -0,0 +1,29 @@
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE neosidekick_aiassistant_domain_model_automationsconfiguration DROP generateemptyimagealttextsonpublication, DROP generateemptyimagetitletextsonpublication, DROP includeneosidekickbriefinginprompt, DROP seoautomationsapprovalprocess, DROP imagealttextautomationsapprovalprocess, DROP brandguardapprovalprocess, DROP brandguardprompt');
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)